react-instantsearch-hooks/geo-search/sandbox.config.json

```diff
@@ -0,0 +1,6 @@
+{
+  "template": "node",
+  "container": {
+    "port": 1234
+  }
+}
```

Comment:
I'm shipping this config to rely on `parcel` in Codesandbox instead of their internal mechanism that currently has issues with type imports in TypeScript and automatic JSX runtime (no explicit `import React from 'react'`).